Added Search Bar

Arushisharma1610 opened the pull request:
**Added Search bar**

added new search bar with graphics.
kindly let me know for any updation



Issue no : #511

Arushisharma1610 commented:
@jfmartinz kindly tell why the checks are failed

jfmartinz commented:
> @jfmartinz kindly tell why the checks are failed

Don't worry about it. Can you give me the output of the changes (give me screenshot)? thanks

Arushisharma1610 commented:
@jfmartinz as the search bar i added is in graphic from so its functionality can only we visible in the form of video ...which i have added below...please let me know in case it is not opening.
Thankyou :)

https://github.com/jfmartinz/ResourceHub/assets/93020354/5d4c5cb5-34e2-415d-bbb3-2402fdce3bf1

jfmartinz commented:
Is the search functionality still working?

Arushisharma1610 commented:
@jfmartinz yes it is working properly

Arushisharma1610 commented:
@jfmartinz hello please review this

jfmartinz commented:
Conflict

Arushisharma1610 commented:
@jfmartinz please review it now ..and let me know if any issue is still their

Arushisharma1610 commented:
@jfmartinz please review it again

Arushisharma1610 commented:
please review it

jfmartinz commented:
Can I see the output? we're still trying to enebale the preveiw in vercell

Arushisharma1610 commented:
https://github.com/jfmartinz/ResourceHub/assets/93020354/4e62451c-ce90-4f11-9c8d-4d9110f174cf

Arushisharma1610 commented:
@jfmartinz please review it i have attached the output also

jfmartinz commented:
I mean search for specific category
